src/preprocessing.py

```diff
@@ -81,9 +83,61 @@ def preprocessing(df, country_metric_df):
         else:
             usagebase_mapper[variable] = 2
     df['usage_base'] = df['usage_base'].replace(usagebase_mapper)
-    
+
+    # Categories
+    categories_list = ['AI & Data Science',
+                        'Business & Technology',
+                        'Environment & Weather',
+                        'Finance & Banking',
+                        'Food, Health & Medicine',
+                        'GeoInformatics & Navigation',
+                        'Government & Public Services',
+                        'Health Science & Medicine',
+                        'Information & Science',
+                        'Justice & Public Safety',
+                        'Logistics & Infrastructure',
+                        'Natural Resources & Energy',
+                        'News & Media',
+                        'None',
+                        'Religion & Spirituality',
+                        'Research & Education',
+                        'Sales & Marketing',
+                        'Security & Technology',
+                        'Skills & Career Development',
+                        'Social Media & Technology',
+                        'Software & Services',
+                        'Sports & Entertainment',
+                        'Transportation & Automobile',
+                        'Work & Personal Life',
+                        'eCommerce & Trade']
+
+    category_enc = OneHotEncoder(handle_unknown="ignore", sparse=False)
+    cat = df[['column']]
```

Comment:
I got an error trying to run this, is there a column named column? also, it would be great if you could put the example run command on top docstring